package.json

```diff
@@ -10,23 +10,21 @@
   "packageManager": "yarn@3.2.4",
   "private": true,
   "scripts": {
+    "allure": "./integration-tests/run-allure.sh",
+    "allure:stop": "docker-compose -f integration-tests/docker-compose.allure.yaml down",
     "build": "tsc --noEmit && ts-node ./esbuild.config.ts",
     "lint": "prettier . --check || exit 1 ; eslint . --max-warnings=0",
     "lint:fix": "prettier . --write ; eslint . --fix",
     "postinstall": "husky install",
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "test": "jest -c jest.config.ts",
     "test:cov": "jest -c jest.config.ts --coverage",
-    "test:integration": "yarn test:cleanReports && jest --setupFiles ./.integration.test-${ENV:-build}.env -c jest.integration.config.ts -i",
-    "test:generateEmailReport": "yarn test:stopReportContainer && docker run -d --rm --name allure -p 5050:5050 -v ${PWD}/allure-results:/app/allure-results -v ${PWD}/allure-reports:/app/default-reports frankescobar/allure-docker-service && mv ${PWD}/allure-reports/emailable-report-allure-docker-service.html ${PWD}/allure-reports/TxMA-Team2-Test_Report-$(date +%d-%m-%Y).html",
-    "test:stopReportContainer": "docker stop allure",
-    "test:showAllureReport": "allure serve",
-    "test:cleanReports": "rm -f -r allure-results/ && rm -f -r allure-report/ && rm -f -r allure-reports",
-    "createTestTicket": "ZENDESK_RECIPIENT_EMAIL=$0 FIXED_DATA_REQUEST_DATE=$1 FIXED_SUBJECT_LINE=$2 OVERRIDE_EVENT_IDS=$3 DATA_PATHS=$4 ts-node scripts/createAndApproveZendeskTicket.ts"
+    "test:integration": "jest -c integration-tests/jest.integration.config.ts -i",
+    "test:integration:dev": "jest --setupFiles '<rootDir>/.env' -c integration-tests/jest.integration.config.ts -i",
```

Comment:
Should we have a script for running locally against other environments using their respective env files, instead of replacing the values in one file? There are quite a few of them. Also, minor point - do we want the env file to live under the integration-tests folder or under the project root directory.

Reply:
No, to run locally against other environments you just log in under that environment. The vars are then automatically picked up from AWS.

If you mean the .env file, then I would also say no as the point of this change is to move away from that. Going forwards we should have only a rare use for that. I left it in only to be used in the rare circumstances it might prove useful, such as testing against a dev stack.